### ISSUE TITLE
Add test button to logging endpoint

### ISSUE THREAD

**Issue by n313893254** [rancher/rancher#17226]
UI for https://github.com/rancher/rancher/issues/15380

**Comment by davidnuzik** [comment]
@n313893254 could you briefly describe what the new logging TEST button is doing? Is it just validating the form or YAML? For example I can put in incorrect information such as wrong endpoint for fluentd and the test still passes.

**Comment by n313893254** [comment]
Verify that the endpoint can be connected，it also validating the form or YAML before post a test action to the backend.

Because of UI just verify endpoints format, so it needs a test button to try endpoint can be connected.

**Comment by davidnuzik** [comment]
@n313893254 is it just checking to see if the port is open? For example I do NOT have fluentd running on my machine yet can successfully test 159.65.64.22:24224 even though there is nothing listening on that port and no fluentd.
![image](https://user-images.githubusercontent.com/45179589/51995647-50ad9100-2470-11e9-9773-da4ddb4ab9d8.png)

**Comment by n313893254** [comment]
@davidnuzik Yes, it is like the notifier test button to check the port before save, if the user needs it.

**Comment by davidnuzik** [comment]
Version: Master 2.2 (1/31/19)
A TEST button was added to validate that a machine is reachable. This button appears to be working correctly. You will get a message that describes what went wrong if the machine cannot be reached for example: `create topic message failed: dial tcp 192.168.99.101:9200: i/o timeout` with kafka and `request to logging target https://192.168.99.101:9200/david-do-daily-2019-01-31/_doc failed: Post https://192.168.99.101:9200/david-do-daily-2019-01-31/_doc: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)` with elasticserach. Each message indicates a timeout if the host:port is not reachable. When your configuration is correct (I tested with fluentd) and the endpoint is reachable the Test will succeed with a message indicating success on the button itself.
![image](https://user-images.githubusercontent.com/45179589/52074109-5ecde100-2546-11e9-9fcf-32deeb4e96f0.png)
For fluentd I noticed I can plug anything in the endpoint field and the test always passes so I entered this issue around this: #17791
